src/sonic-bgpcfgd/tests/data/sonic-cfggen/bgpd.conf.j2/all.conf

```diff
@@ -29,14 +33,15 @@ ip prefix-list PL_LoopbackV4 permit 55.55.55.55/32
 !
 ipv6 prefix-list PL_LoopbackV6 permit fc00::/64
 !
-ip prefix-list LOCAL_VLAN_IPV4_PREFIX seq 5 permit 10.10.10.1/24
+ip prefix-list LOCAL_VLAN_IPV4_PREFIX seq 10 permit 10.10.10.1/24
```

Comment:
is this intended?

Reply:
The it is an artefact of the order change due to the elemination of natsorted. The [bgpd.conf.j2](https://github.com/Azure/sonic-buildimage/blob/master/dockers/docker-fpm-frr/frr/bgpd/bgpd.main.conf.j2#L22) user the loop index and multiplies it by 5 and so the seq would depend on the index of the IP address